data/nmstate/003-operator.yaml

```diff
@@ -31,6 +31,13 @@ spec:
           - --v=production
           image:  {{ .NMStateHandlerImage }}
           imagePullPolicy: {{ .ImagePullPolicy }}
+          resources:
+            requests:
+              cpu: "100m"
```

Comment:
60m ?

Reply:
Calling `nmstatectl` Python application is expensive. I'm afraid even 60m will not be enough. Let's keep it on 100 for now.